qemuesesc_barebone.cpp

```diff
@@ -31,6 +31,26 @@ extern "C" void QEMUReader_goto_sleep(void *env);
 extern "C" void QEMUReader_wakeup_from_sleep(void *env);
 extern "C" void esesc_set_timing(uint32_t fid);
 
+uint64_t last_pc = 0;
+uint8_t ctrl_flag = 0;
+
+uint8_t checkpc(uint64_t pc){
+		if (ctrl_flag == 1){
```

Comment:
If the last instruction is ctrl, we need to enter this if.
ctrl_flag means that the last instruction is ctrl.